study.md

```diff
@@ -87,7 +91,13 @@ Create the following movies with the given attributes.
 | 2 | The Core | 5.4 |
 
 ```ruby
-# your answer here
+
+battlefield_earth = Movie.create(id: 0, name: "Battlefield Earth", rating: "2.4");
```

Comment:
We don't want to set `id`s because a database table with `id` as a primary key column will do it for us.  The value of the `rating` attribute shouldn't have quotation marks around it because it should be a number and not a string.